packages/gestalt/src/Contents.js

```diff
@@ -564,3 +582,14 @@ export default class Contents extends React.Component<Props, State> {
     );
   }
 }
+
+export default function Contents(props: OwnProps): React.Node {
```

Comment:
Instead  of having to wrap this to get the hook, could it reference the Context directly?

Reply:
I could export a render prop context consumer and use that instead for class components. Do you think that's something we should export externally as well?